hack/update-clientgen.sh

```diff
@@ -8,7 +8,7 @@ CODEGEN_PKG=${CODEGEN_PKG:-$(cd ${SCRIPT_ROOT}; ls -d -1 ./vendor/k8s.io/code-ge
 verify="${VERIFY:-}"
 
 for group in ${API_GROUP_VERSIONS}; do
-  bash ${CODEGEN_PKG}/generate-groups.sh "client,lister,informer" \
+  GOFLAGS="" bash ${CODEGEN_PKG}/generate-groups.sh "client,lister,informer" \
```

Comment:
I'm not sure what sort of template you used for generating those, but this should be there. The Image builder uses `GOFLAGS="-mod=vendor", so the client gen complains about its dependencies.

Reply:
/hold
this is suspicious

Reply:
The same change is in `update-clientgen.sh` BTW :)

Reply:
this script is inspired by https://github.com/openshift/client-go/blob/master/hack/update-codegen.sh. Can we find out why we don't need GOFLAGS="" there?

Reply:
Fixed with other method.